Automattic/jetpack
# Launchpad: Add fullscreen launchpad skip feature

Related to https://github.com/Automattic/wp-calypso/issues/76876

## Proposed changes:
This PR adds a new state option for the launchpad checklist, `skipped`. This way we are able to _not_ redirect users to the fullscreen Launchpad every time they try to go to the customer home.
This feature is gonna be triggered by Calypso in a separate PR (https://github.com/Automattic/wp-calypso/pull/80661)

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:






*  Apply this PR to your sandbox
* Access calypso.live from https://github.com/Automattic/wp-calypso/pull/80661
* Create a site with `/start`
* You will be redirected to fullscreen Launchpad
* Click on the `Don't show this again` button
* You should be redirected to the customer home
